md_api/src/main/java/com/wipro/ats/bdre/md/api/HaltJob.java

```diff
@@ -35,6 +38,10 @@
  * Created by arijit on 12/8/14.
  */
 public class HaltJob extends MetadataAPIBase {
+    public HaltJob() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
 
     private static final Logger LOGGER = Logger.getLogger(HaltJob.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)